src/Propel/Generator/Util/QuickBuilder.php

```diff
@@ -343,11 +382,11 @@ public function getSQL()
     }
 
     /**
-     * @param string[]|null $classTargets
+     * @param array $classTargets
      *
      * @return string
      */
-    public function getBuildName($classTargets = null)
+    public function getBuildName(array $classTargets = []): string
```

Comment:
This accepted null so far.
I am not against making this a bit more strict - but we need to be careful about either the BC approach or to clearly state this in the release notes for the next release coming out.

Reply:
Fixed.